query-service/src/main/resources/configs/common/application.conf

```diff
@@ -82,6 +82,18 @@ service.config = {
       requestHandlerInfo = {
         tenantColumnName = tenant_id
         startTimeAttributeName = "EVENT.startTime"
+#         additionalTenantFilters = [
```

Comment:
since it's already under a table scope, I think we not need add another scope here. 
I don't think we need to further extend it to query scope(Its not even available in query as we discussed)